### PR TITLE
[DO NOT MERGE] Feature/jshint

### PR DESCRIPTION
**Pull request by kula1922**
Befor this pull request will merge there is a need update ralph installation process.
- Created ventor in static directory
- Created configs for nps grunt bower and karma
- Created first tests in angular
- Clean up static files

### CODE REVIEW

**Comment by ar4s** [comment]
Nps? To much nps :) probably you mean npm.

File: Gruntfile.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+                        'bower_components/angular-resource/angular-resource.min.js',
+                        'bower_components/angular-resource/angular-resource.min.js.map',
+                        'bower_components/angular-route/angular-route.min.js',
+                        'bower_components/angular-route/angular-route.min.js.map',
+                        'bower_components/bootstrap/dist/js/bootstrap.min.js',
+                        'bower_components/bootstrap-table/dist/bootstrap-table.min.js',
+                        'bower_components/jquery/dist/jquery.min.js',
+                        'bower_components/jquery/dist/jquery.min.js.map',
+                        'bower_components/multiple-select/jquery.multiple.select.js',
+                        'bower_components/ng-scrollbar/dist/ng-scrollbar.min.js',
```
Comment (ar4s) [inline Gruntfile.js]: use some wildcards e.g. ``bower_components/angular/*.js`` or ``bower_components/**/*.js``

File: Gruntfile.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                {
+                    expand: true,
+                    flatten: true,
+                    src: [
+                        'bower_components/bootstrap/dist/css/bootstrap.min.css',
+                        'bower_components/bootstrap/dist/css/bootstrap-theme.min.css',
+                        'bower_components/bootstrap-table/dist/bootstrap-table.min.css',
+                        'bower_components/multiple-select/multiple-select.css',
+                        'bower_components/ng-scrollbar/dist/ng-scrollbar.css',
+                        'bower_components/font-awesome/css/font-awesome.min.css'
```
Comment (ar4s) [inline Gruntfile.js]: ditto

File: Gruntfile.js (hunk 1 above)
Comment (kula1922) [inline Gruntfile.js]: There is no way, bower download all sources with anyother *.js fields. Any non-absolute path could  copy more files then we need.

File: Gruntfile.js (hunk 2 above)
Comment (kula1922) [inline Gruntfile.js]: dito

**Comment by mkurek** [comment]
Couldn't merge it and probably we'll try gulp. Closing.

**Pull request closed without merging**